pages/research_software_stories/acts_research_software_story.md

```diff
@@ -0,0 +1,177 @@
+high-energy and nuclear physics communities (including heavy-ion
+experiments).  Contributors come from institutions worldwide.  ACTS is
+actively used in ATLAS, CERN's Experimental Physics R&D projects, and
+various detector R&D studies.  It also attracts researchers developing new
+tracking algorithms for advanced computing architectures by acting as a
+modular test suite and comparison baseline.  This broad adoption has created
+a large, collaborative base of users and developers.
+
+
+## Technical Aspects
+
+ACTS is a high-performance, thread-safe library written in modern C++.  Its
+core components include:
+
+- Geometry description: An optimised detector geometry model for fast
+  navigation.
+
+- Tracking algorithms: Tools for track finding and fitting, seeding, vertex
+  finding, etc.
+
+- Event Data Model (EDM): A standardised way to represent track data.
+
+ACTS is optimised for speed, using compile-time polymorphism (templates)
+instead of runtime inheritance to improve performance.  It keeps
+non-optional dependencies to a minimum (mainly the Eigen linear algebra
+library).  The toolkit uses CMake for building and runs on Linux with
+multi-core support.  It can integrate with external geometry definitions
+(eg.  ROOT TGeo or DD4hep), making it easier to use in different
+experiments' software frameworks.
+
+
+## Libraries and Systems
+
+- C++20: Leverages modern C++ features for performance and concurrency.
+
+- Eigen: A linear algebra library used for efficient mathematical
+  computations.
+
+- Geometry converters: Interfaces to import detector geometry from ROOT TGeo
+  and DD4hep.
+
+- GPU acceleration: Experimental support for using GPUs to speed up certain
+  computations.
+
+- ML integration: Plugins like Exa.TrkX for GNN based tracking or ONNX for
+  development and testing of ML algorithms for ambiguity resolution, seed
+  filtering etc.
+
+
+## Software Practices
+
+ACTS development follows strict software engineering standards.  The project
+is developed openly on GitHub, and every code submission triggers automated
+CI tests to ensure nothing breaks and that thread-safety is maintained. 
+Developers adhere to coding and contribution guidelines, and use peer
+reviews on merge requests.  Issues are tracked transparently, and frequent
+meetings plus mailing list discussions keep contributors aligned and
+informed.
+
+
+## Community
+
+The ACTS community provides many resources to help onboard newcomers:
+
+- Contribution guidelines: A clear step-by-step guide for submitting code.
+
+- Interactive tutorials: Hands-on guides for configuring and running
+  tracking workflows.
+
+- Example framework: A sample project demonstrating how to use ACTS
+  algorithms in practice.
+
+In addition, regular workshops and training sessions help new members start
+using ACTS in their experiments.  New contributors are often mentored by
+experienced developers, ensuring knowledge transfer and continuous growth of
+the community.
+
```

Comment:
I think links to the contribution guidelines, interactive tutorials and example framework would make this section richer. Also feel free to update the Community title in these software stories and I can update the template to follow.